Weather Alert/CoreData/City+CoreDataProperties.swift

```diff
@@ -16,5 +16,12 @@ extension City {
         return NSFetchRequest<City>(entityName: "City");
     }
 
+    @NSManaged public var deg: NSNumber?
+    @NSManaged public var name: String?
+    @NSManaged public var speed: NSNumber?
+    @NSManaged public var lon: NSNumber?
+    @NSManaged public var lat: NSNumber?
+    @NSManaged public var country: String?
+    @NSManaged public var id: NSNumber!
```

Comment:
@AlexKel This is my doing. I explained the purpose and reason and, by knowing the mock data has id's, set it as unwrapped. Initially it was set as optional in the model but not in the var declaration, so just changing it to be an optional was enough for the sake of understanding the difference.